crates/bevy_animation/src/animation_curves.rs

```diff
@@ -984,3 +992,21 @@ macro_rules! animated_field {
         })
     };
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_animated_field_tuple_struct_simple_uses() {
+        #[derive(Clone, Debug, Component, Reflect)]
+        struct A(f32);
+        let _ = AnimatedField::new_unchecked("0", |a: &mut A| &mut a.0);
```

Comment:
I'm not super familiar with all the animation machinery, but for structs with named fields, the field name is used as part of the `EvaluatorId`, which I think identifies what part of the component is being animated? If you're manually constructing an `AnimatedField` (without the macro) you still have to do something like `AnimatedField::new_unchecked("bar", |foo: &mut Foo| &mut foo.bar)` for structs with named fields, so I figured it was fine to maintain the field name requirement, especially since I'm not quite sure what the `EvaluatorId` would be without being given the field name.

Reply:
@yonzebu That is correct; in turn, it's important that the `EvaluatorId` incorporates the field index so that it can be known when two animatable properties are "the same" (this is used to dispatch the animation evaluators).